newinstaller/style.css

```diff
@@ -0,0 +1,221 @@
+
+/* Import fonts Noto Sans and Montserrat */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Noto+Sans:ital,wght@0,100..900;1,100..900&display=swap');
+
+/* Variable Declaration*/
```

Comment:
Remove comment for obvious declarations.

Reply:
Keeps this comment for clarification.